PC/python3.def

```diff
@@ -224,7 +224,6 @@ EXPORTS
   PyExc_PermissionError=python37.PyExc_PermissionError DATA
   PyExc_ProcessLookupError=python37.PyExc_ProcessLookupError DATA
   PyExc_RecursionError=python37.PyExc_RecursionError DATA
-  PyExc_RecursionErrorInst=python37.PyExc_RecursionErrorInst DATA
```

Comment:
This line means `PyExc_RecursionErrorInst` is part of the stable ABI and shouldn't be removed at all. See [PEP 384](https://www.python.org/dev/peps/pep-0384/) for more information about the stable ABI.